index.html

```diff
@@ -19,7 +19,13 @@ <h1>Welcome to the student showcase!</h1>
     </div>
     <div class="content">
         <div>ONE</div>
-        <div>TWO</div>
+        <div>
+          <a href="https://avatars0.githubusercontent.com/u/36180706?s=400&u=71f427f947a5a04fecb935ba22a27d5fe58239c7&v=4">Marquis Patton's Profile</a>
```

Comment:
This is how we add a _link_/anchor tag, but make sure that you're adding an `<img/>` instead